src/ifvc.c

```diff
@@ -246,6 +248,19 @@ void buildIfVc(void) {
             if (IfNext < IfPt + 1)
                 IfNext = IfPt + 1;
 
+            /* don't retrieve any further info if MAX_IF is reached
+             */
+            if ( IfDescEp >= &IfDescVc[ MAX_IF ] ) {
+                my_log(LOG_DEBUG, 0, "Too many interfaces, skipping all since %s", IfPt->ifr_name);
+                break;
+            }
+
+            /* don't retrieve any info from non-IPv4 interfaces
+             */
+            if ( IfPt->ifr_addr.sa_family != AF_INET ) {
+                my_log(LOG_DEBUG, 0, "Interface is not AF_INET, skipping %s (family %d)", IfPt->ifr_name, IfPt->ifr_addr.sa_family);
+                continue;
```

Comment:
Skipping interfaces without IPv4 address would mean igmpproxy itself would not see this interface. Why such change is needed? Previously these interfaces was normally processed just marked as non-IPv4.